src/parsec_service/install_parsec_linux.md

```diff
@@ -24,23 +25,46 @@ git clone https://github.com/parallaxsecond/parsec.git
 cargo install --features $DESIRED_FEATURES --path parsec
 ```
 
-Copy and adapt the [configuration](configuration.md) you want to use.
+Copy and adapt the [configuration](configuration.md) you want to use. For a secure deployment, make
+sure to activate the `log_error_details` option and to use a `trace` log level.
 
 ```
 cp parsec/config.toml config.toml
 ```
 
+Create the Parsec socket directory.
+
+```
+mkdir /tmp/parsec
+```
+
+In a deployment **without an Identity Provider**, create the `parsec-clients` group and set the
+correct permissions on the socket folder. Mutually trusted Parsec Clients will need to be in that
```

Comment:
Yes, the idea is that the clients trust each other (i.e. they won't use each other's identities) - this is needed because if you have multiple clients and no identity provider, you *must* have some sort of trust in the other clients

Reply:
To add to what Ionut said, this is directly coming from operational mitigation 10 of the service TM:

> A set of mutually trusted clients has restricted read-write access to the service IPC endpoint.

Since any client in that group can access the keys of the other ones, we seemed to think that the trust should be in between the clients.

Reply:
Added the following paragraph:

```
+Similarly to the threat model, this guide proposes different alternatives in case an Identity Provider
+is available or not. The role and description of an Identity Provider in Parsec is described in the
+[System Architecture](https://parallaxsecond.github.io/parsec-book/parsec_service/system_architecture.html) page.
+Currently, Parsec does not support integration with any Identity Provider. To securely install Parsec, please
+follow the steps of deployment **without an Identity Provider**.
```